compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParserErrorHandler.java

```diff
@@ -790,143 +791,6 @@ public BallerinaParserErrorHandler(AbstractTokenReader tokenReader) {
         super(tokenReader);
     }
 
-    /**
-     * @deprecated This method is no longer used for its original purpose and has not been maintained for a while.
-     */
-    @Override
-    @Deprecated
-    protected boolean isProductionWithAlternatives(ParserRuleContext currentCtx) {
```

Comment:
Shall we remove this from abstractErrorHandler and XMLParserErrorHandler as well?